### ISSUE TITLE
Symbol’s function definition is void: orgit-link-set-parameters

### ISSUE THREAD

**Issue by gsnewmark** [magit/orgit#17]
When trying to use the latest version of this package (`20161104.1823`) I get the following error:
```
Compiling file /home/gsnewmark/.emacs.d/elpa/orgit-20161104.1823/orgit.el at Sat Nov  5 14:25:17 2016
Entering directory ‘/home/gsnewmark/.emacs.d/elpa/orgit-20161104.1823/’

Compiling no file at Sat Nov  5 14:25:17 2016
Error (bytecomp): Symbol’s function definition is void: orgit-link-set-parameters
```

Emacs version is `"GNU Emacs 25.1.1 (x86_64-unknown-linux-gnu, GTK+ Version 3.20.9) of 2016-09-18"`.

The same error is reproduced when I try to manually byte-compile from source:
```
orgit.el:66:1:Error: Symbol’s function definition is void: orgit-link-set-parameters
```

**Comment by tarsius** [comment]
I cannot reproduce this using `emacs -Q --batch -L . -L ../dash -L ../magit/lisp -L ../with-editor -f batch-byte-compile orgit.el`. The function in question is defined at the top of the file before anything else. It was added in eddf0fb18cb8bd4186723d06020b2dd3c184992c.

**Comment by gsnewmark** [comment]
Thanks, batch compilation works for me too, so it's probably a problem with order of packages loading, I'll check my config.

**Comment by tarsius** [comment]
Fixed with #18.
